python/lsst/obs/subaru/ingest.py

```diff
@@ -105,7 +105,7 @@ def translate_pointing(self, md):
 
         try:
             dateobs = md.getScalar("DATE-OBS")
-            m = re.search(r'(\d{4})-(\d{2})-(\d{2})', dateobs)
+            m = re.search(r'(\d{4})-(\d{2})-(\d{2})', dateobs) # noqa
```

Comment:
Again does this really need an exception?  Is it complaining about no spaces around the operators?